lib/chef-acceptance/shellout_builder.rb

```diff
@@ -0,0 +1,48 @@
+module ChefAcceptance
+  class ShelloutBuilder
```

Comment:
This is nice. But I think attributes like `:recipe, :chef_config_file, :dna_json_file`  make this class very specific to be able to run a chef-client command. I do not see how we will re-use this code from somewhere else which makes me ask myself: do we really need a class for this purpose? 

I think this can simply be done in a method in the `ChefRunner`.